### ISSUE TITLE
Content Mismatch

### ISSUE THREAD

**Issue by sandarumk** [systers/PCSA-Android#431]
All the content on this app comes from this document. 
https://drive.google.com/file/d/0B8cm8EBZg0GaQy1CUkRUOXhWZUE/view
There are few places that the content in the document does not match with the content in the app. In this task what you have to do is go through the app and find the inconsistencies between the content and the app and list them.

**Comment by ashu-dadhich** [comment]
Can I attach a document listing out various discrepancies so that new-comers can work on them?

**Comment by sandarumk** [comment]
If there is a document it will be hard to assign and solve. instead you can add them here as a task list.

**Comment by sandarumk** [comment]
Shall we give this to somebody new?

**Comment by ashu-dadhich** [comment]
A new-comer might not be that familiar with the app and this might lead some discrepancies unreported or they might point out details that might not even be necessary.
So are we assigning this to a new-comer or shall I go ahead?

**Comment by sandarumk** [comment]
This is simply validating the content by going through the app. He/she do not have to be familiar with the app. And by doing this they may get the chance to familiarize with the app. If they have any unreported stuff we can make improvements in the future. And we currently have only few issues for new comers. So we are going to keep this as a new comer task.

**Comment by ashu-dadhich** [comment]
Yeah it would be a great idea. I didn't thought about that in this way plus it would solve our problem of not having more issues for new comers.

**Comment by codeb1ooded** [comment]
@sandarumk Can I work on this issue? Also I wanted to know that do you only want to list inconsistencies for now? Or also solve them alongside?

**Comment by sandarumk** [comment]
Let's talk about fixing later. For now, what we need is a list of inconsistencies.

**Comment by ashu-dadhich** [comment]
@sandarumk should I make a list of inconsistencies just like the lint list?
